File: haohaninfo/回測範例/23.py
#取I020，依照逗點分隔，並將分隔符號去除
I020 = [ line.strip('\n').split(",") for line in open('Futures_20170815_I020.csv')][1:]

#取得特定時間前的價格資料
p= [int(line[4]) for line in I020 if  int(line[0])<9000000]
#取得特定時間前的總量資料
q= [int(line[6]) for line in I020 if int(line[0])<9000000]

print p[-1],q[-1]

File: haohaninfo/交易回測案例/23.R
setwd("資料夾路徑")
I020 <- read.csv('./Futures_20170815_I020.csv')
p1 <- subset(I020,INFO_TIME>8590000 &INFO_TIME<=9000000)
c(p1[1,]$Match_Price,max(p1$Match_Price),min(p1$Match_Price),p1[nrow(p1),]$Match_Price,sum(p1$QTY))